API/server.js

```diff
@@ -0,0 +1,18 @@
+import express from 'express';
+import dotenv from 'dotenv';
+import moviesRoutes from './routes/moviesRoutes';
+const app = express();
+dotenv.config();
+
+app.use('/movies', moviesRoutes);
+
+app.get('/', (req, res) => {
+  res.send('Main').status(200);
+});
+
+app.get('/*', (req, res) => {
```

Comment:
It's an API server not an web server. There shouldn't be a need to redirect anywhere. OK to remove this entire redirect

Reply:
It is a type of web server, fwiw. But it should not really be redirecting anywhere. Removed this.